14_functions_lab/01.py

```diff
@@ -1,9 +1,17 @@
-import unittest
+#!/usr/bin/python
 
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
+def my_sum(*args):
+    for arg in args:
+        if not type(arg) == int:
+            return "input numbers only"
```

Comment:
1. Python has a function called `any` which makes writing loops like this easier. This line does the same thing:

```
if any(type(thing) is not int for thing in things):
  return "input numbers only"
```

2. It's better to write functions that always return the same type. In such cases it's better to raise an exception